TS.fsx

```diff
@@ -784,6 +784,7 @@ module Emit =
         for e in tagNameToEleName do
             if iNameToIDependList.ContainsKey e.Value && Seq.contains "HTMLElement" iNameToIDependList.[e.Value] then
                 Pt.Printl "\"%s\": %s;" (e.Key.ToLower()) e.Value
+        Pt.Printl "[index: string]: HTMLElement;"
```

Comment:
I see. To fix that, instead of placing the index signature directly inside `ElementTagNameMap`, I have created another interface (known as `IndexedElementTagNameMap`) that extends `ElementTagNameMap` and moved the index signature there.

The change is now in commit f8c3f19e3143d045ba077ced23cda14b5ba37924.